docs_src/examples/Ch-OSImageWithEdgeX.md

```diff
@@ -0,0 +1,624 @@
+# Creating an EdgeX OS Image
+
+This document walks you through creating an OS image that is preloaded with an EdgeX stack. We use Ubuntu Core as the OS, which is optimized for IoT and allows secure deployment of applications. We build the image using the default tooling and the snapped versions of EdgeX components. Since snaps receive automatic and transactional updates, the components stay up to date reliably with the latest security and bug fixes.
+
+This guide is divided into three chapters:
+
+- Creating an image with EdgeX components, using default configurations
+- Overriding basic service configurations
+- Replacing default service configurations
```

Comment:
```suggestion
- A. Creating an image with EdgeX components, using default configurations
- B. Overriding basic service configurations
- C. Replacing default service configurations
```

Reply:
I don't want to make this change because it doesn't look very good to add letters in an unordered list. Sadly, Mkdocs doesn't support ordered list with letters. Also, the text doesn't match the chapter titles.

I'll keep this open and revisit it when chapter C is ready.